data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/IsolationLevel.java

```diff
@@ -0,0 +1,33 @@
+package org.opensearch.dataprepper.plugins.kafka.configuration;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Map;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public enum IsolationLevel {
```

Comment:
@lhooss , Thank you for updating to an enum.

Please also add some unit tests for this enum similar to these:

https://github.com/opensearch-project/data-prepper/blob/af5dce7cd019a577aa6ae64a8081cd9beb787dc2/data-prepper-plugins/aggregate-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/RateLimiterModeTest.java#L26